libvast/src/command.cpp

```diff
@@ -131,6 +131,10 @@ bool init_config(caf::actor_system_config& cfg, const command::invocation& from,
   };
   // Merge all CLI settings into the actor_system settings.
   merge_settings(from.options, cfg.content);
+  // Allow users to use `system.verbosity` to configure console verbosity.
+  if (auto value = caf::get_if<caf::atom_value>(&from.options,
+                                                "system.verbosity"))
+    cfg.set("logger.console-verbosity", *value);
```

Comment:
One additional thought: how about issuing a warning when the requested log level is higher than the compile-time maximum? A silent fallback might be an opportunity for confusion.